bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/views/report/list/ReportListView.java

```diff
@@ -49,16 +74,76 @@
     private IProject currentProject;
     private ShowFilterConfigurationDialog showfilterconfigurationdialog;
 
+    private Label message;
+    private Composite client;
+
     public ReportListView() {
     }
 
     public void createPartControl(Composite parent) {
 
         this.parent = parent;
 
-        parent.setLayout(new GridLayout(1, true));
+        client = new Composite(parent, parent.getStyle());
+        GridLayoutFactory.fillDefaults().numColumns(2).applyTo(client);
+
+        Button analyzeBtn = new Button(client, SWT.PUSH);
+        analyzeBtn.setToolTipText("Reanalyze current file");
+        analyzeBtn.setImage(
+                AbstractUIPlugin.imageDescriptorFromPlugin(Activator.PLUGIN_ID, "icons/analyze.png").createImage());
+        analyzeBtn.addMouseListener(new MouseListener() {
+
+            @Override
+            public void mouseUp(MouseEvent e) {
+                // if (currentFilename == null || currentProject == null)
+                // return;
+                IEditorPart editor = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage()
+                        .getActiveEditor();
+                if (editor == null)
+                    return;
+                IEditorInput input = editor.getEditorInput();
+                if (input == null)
+                    return;
+                Path file = null;
+                IProject proj = null;
+                try {
+                    file = ((FileEditorInput) input).getPath().toFile().toPath();
+                    proj = ((FileEditorInput) input).getFile().getProject();
+                } catch (Exception ex) {
+                    return;
+                }
+                if (file == null)
+                    return;
+                AnalyzeJob analyzeJob = new AnalyzeJob(proj, file);
+                PlistParseJob plistParseJob = new PlistParseJob(proj);
+                analyzeJob.setRule(proj);
+                plistParseJob.setRule(proj);
+                final IProject fproj = proj;
+                plistParseJob.addJobChangeListener(new JobDoneChangeListener() {
+                    @Override
+                    public void done(IJobChangeEvent event) {
+                        CodeCheckerContext.getInstance().refresAsync(fproj);
+                    }
+                });
+
+                analyzeJob.schedule();
+                plistParseJob.schedule();
+            }
+
+            @Override
+            public void mouseDown(MouseEvent e) {
+                ; // noop
+            }
+
+            @Override
+            public void mouseDoubleClick(MouseEvent e) {
+                ; // noop
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641621)